TestingDangerApp/ViewController.swift

```diff
@@ -1,26 +1,37 @@
-//
-//  ViewController.swift
-//  TestingDangerApp
-//
-//  Created by PicPay Eduardo on 7/23/19.
-//  Copyright © 2019 PicPay Eduardo. All rights reserved.
-//
-
 import UIKit
 
 class ViewController: UIViewController {
-    lazy var label: UILabel = {
+    lazy var label:UILabel = {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/fbeaccc4035030f034a16e6ce55b9e78fdde932e/TestingDangerApp/ViewController.swift#L4"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>ViewController.swift:4</code></td>
    </tr>
  </tbody>
</table>